openshift/sso74-x509-configmap.yaml

```diff
@@ -126,6 +126,7 @@ objects:
                   <logger category="org.keycloak.events" use-parent-handlers="false">
                       <level name="DEBUG"/>
                       <handlers>
+                          <handler name="CONSOLE"/>
```

Comment:
Is there any way we can tighten up this code? It's a little wordy IMHO. ;-)

Reply:
Hahaha 🥇